packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -75,6 +75,9 @@ export default function pluginContentPages(
             path: permalink,
             component: source,
             exact: true,
+            modules: {
+              config: `${context.generatedFilesDir}/docusaurus.config.js`,
```

Comment:
```suggestion
              config: `@generated/docusaurus.config`,
```

Reply:
Actually can't commit it directly to your PR because I don't have write access @anshulrgoyal , can you apply the suggestion so that I can merge?